apis/cosmos-reducers.js

```diff
@@ -622,7 +619,7 @@ export function transactionReducer(transaction) {
           transaction
         ),
         timestamp: transaction.timestamp,
-        memo: transaction.tx.body.memo,
+        memo: transaction.tx.value.memo,
```

Comment:
no

Reply:
```suggestion
        memo: transaction.tx.body.memo,
```

Reply:
I tested with both e-Money and Cosmos Hub :shrug: